_docs/xcm/hrmp-channel.md

```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
 ```
 
 ## Turing Staging
-Parachain sovereign account: 5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y
+The below details can be viewed on [Rococo Subscan](https://rococo.subscan.io/parachain/2114).
+
+Sovereign Account (Rococo V1): `5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y`
+
+Sovereign Account (Turing Staging): `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`
+
+Sovereign Account (Parachain): `0x7369626c42080000000000000000000000000000000000000000000000000000`
```

Comment:
Does this line mean the Sovereign account on Parachain?

It should be `0x7061726142080000000000000000000000000000000000000000000000000000`.

Reply:
Could you explain how was this value calculated?

Reply:
It should be the sibling account on another parachain such as Shiden, so `0x7369626c42080000000000000000000000000000000000000000000000000000` is correct.